Doc/library/ftplib.rst

```diff
@@ -140,17 +159,30 @@ FTP objects
 
    .. method:: FTP.connect(host='', port=0, timeout=None, source_address=None)
 
-      Connect to the given host and port.  The default port number is ``21``, as
-      specified by the FTP protocol specification.  It is rarely needed to specify a
-      different port number.  This function should be called only once for each
-      instance; it should not be called at all if a host was given when the instance
-      was created.  All other methods can only be used after a connection has been
-      made.
-      The optional *timeout* parameter specifies a timeout in seconds for the
-      connection attempt. If no *timeout* is passed, the global default timeout
-      setting will be used.
-      *source_address* is a 2-tuple ``(host, port)`` for the socket to bind to as
-      its source address before connecting.
+      Connect to the given host and port.
+      This function should be called only once for each instance;
+      it should not be called if a *host* argument was given
+      when the :class:`FTP` instance was created.
+      All other :class:`!FTP` methods can only be called
+      after a connection has successfully been made.
+
+      :param str host:
+         The host to connect to.
+
+      :param int port:
+         The TCP port to connect to.
+         If no port number is provided, port ``21`` is used,
+         as specified by the FTP protocol specification.
+         It is rarely needed to specify a different port number.
+
+      :param timeout:
+         A timeout in seconds for the connection attempt.
+         If not specified, the global default timeout setting will be used.
```

Comment:
?

(If so, also update line 104)
```suggestion
         A timeout in seconds for the connection attempt
         (default: the global default timeout setting).
```

Reply:
Yes, I like these suggestions better than the existing wording. I'll apply to L104 as well.